srv/pom.xml

```diff
@@ -42,6 +42,12 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>com.sap.cds</groupId>
+			<artifactId>cds-feature-hana</artifactId>
```

Comment:
This dependency is already part of the starter package `cds-starter-cloudfoundry`. No need to add it explicitly again.